pyproject.toml

```
@@ -4,17 +4,17 @@
 
 [project]
 name = "brainscore_vision"
-version = "2.0"
+version = "2.1.0"
```

```suggestion
version = "2.1"
```
let's keep it simple